apps/dashboard/src/components/primitives/tag-input.tsx

```diff
@@ -27,10 +27,16 @@ const TagInput = forwardRef<HTMLInputElement, TagInputProps>((props, ref) => {
   }, [value]);
 
   const addTag = (tag: string) => {
+    const newTag = tag.trim();
+    if (newTag === '') {
+      return;
+    }
+
     const newTags = [...tags, tag];
     if (new Set(newTags).size !== newTags.length) {
       return;
     }
+
```

Comment:
Accidentally brought across from the other PR

```suggestion
const newTags = [...tags, tag];
    if (new Set(newTags).size !== newTags.length) {
      return;
    }
```

Reply:
Thanks! fixed